daemon/lokinet-bootstrap.cpp

```diff
@@ -28,6 +29,31 @@ namespace
     std::cout << msg << std::endl;
     return 1;
   }
+
+  template <typename Map_t>
+  int
+  print_help(std::string exe, const Map_t& urls)
+  {
+    std::cout << "Lokinet bootstrap.signed fetchy program thing" << std::endl;
+    std::cout << "Downloads the initial bootstrap.signed for lokinet into a local file from a "
+                 "default or user defined server via the reachable network"
+              << std::endl
+              << std::endl;
```

Comment:
Should just use \n's, `<< std::endl` is... misunderstood.  It's equivalent to `<< "\n" << std::flush;` which forces a flush and is almost always unnecessary (e.g. stdout is already flush-on-newline when output is a tty).

You maybe could also just literal string the whole thing to make it a bit nicer:

```C++
  std::cout << R"(Lokinet bootstrap.signed fetchy program thing

Downloads the initial bootstrap.signed for lokinet into a local file from a default or user
defined server via the reachable network.

Usage: )" << exe << R"( [bootstrap_url [output_file]]

bootstrap_url can be a named value in which we will use a predefined url that
holds a bootstrap file

)";
```
etc.

Reply:
(People commonly think `std::endl` means "do the right thing on (windows|other archaic systems)" but it doesn't; that magic happens with "\n" in the first place.  `std::endl` just says "print \n and then force a flush").